src/piton/extractors/csv.py

```
@@ -140,48 +140,48 @@ def run_csv_converters(
 
     target = EventCollection(target_location)
 
-    files_per_converter = [0 for _ in converters]
+    files_per_converter = [0 for _ in extractors]
 
     to_process = []
 
     for root, dirs, files in os.walk(source_csvs):
         for name in files:
             full_path = os.path.join(root, name)
             relative_path = os.path.relpath(full_path, source_csvs)
-            matching_converters = [
+            matching_extractors = [
                 (i, a)
-                for i, a in enumerate(converters)
+                for i, a in enumerate(extractors)
                 if (
                     str(relative_path).startswith(a.get_file_prefix() + ".csv")
                     or str(relative_path).startswith(a.get_file_prefix() + "/")
                 )
             ]
-            if len(matching_converters) > 1:
+            if len(matching_extractors) > 1:
                 print(
                     "Multiple converters matched?",
                     full_path,
-                    matching_converters,
+                    matching_extractors,
                 )
                 print(1 / 0)
-            elif len(matching_converters) == 0:
+            elif len(matching_extractors) == 0:
```

For consistency, we probably want to have print("No converters matched", full_path, matching_extractors) after this line?

---

Yeah, this should throw an exception as well with context.